masterlist.yaml

```
@@ -3183,42 +3183,111 @@ plugins:
       - crc: 0x3D0FE375
         util: 'TES4Edit v4.0.4b'
 
-  - name: 'EVE_KnightsoftheNine.esp'
+  - name: '(EVE_.*|Armamentium female|ArtifactsFemaleArmor|Fransfemale)\.esp'
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/24078/'
         name: 'EVE HGEC Eyecandy Variants Expansion'
+  - name: 'EVE_(StockEquipmentReplacer|ShiveringIslesEasterEggs|KnightsoftheNine)\.esp'
+    url:
       - link: 'https://www.nexusmods.com/oblivion/mods/45859/'
         name: 'Seamless - OCOv2'
+  - name: 'EVE_StockEquipmentReplacer.esp'
+    inc:
+      - 'Mart''s Monster Mod.esm'
+      - 'Oscuro''s_Oblivion_Overhaul.esm'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'EVE_StockEquipmentReplacer ESP' ]
+        condition: 'many("EVE_StockEquipmentReplacer(4FCOM|4MMM| for OOO)?\.esp")'
+      - <<: *useVersion
+        type: warn
+        subs: [Martigen''s Monster Mod]
```

This string should be wrapped in single-quotes.

```suggestion
        subs: [ 'Martigen''s Monster Mod' ]
```